app/models/transaction.rb

```diff
@@ -0,0 +1,7 @@
+class Transaction < ApplicationRecord
+  belongs_to :invoice
+
+  validates :credit_card_number, presence: true, length: { maximum: 16 }
+  validates :credit_card_expiration_date, presence: true, format: { with: /\A\d{2}\/\d{2}\z/, message: "should be a MM/YY format"}
```

Comment:
I like how we set ourselves up to test for sad paths with these validations. Great work!

Reply:
for sure! validations and relationships look good!